broker/tasks/cloudfront.py

```diff
@@ -265,30 +275,33 @@ def update_distribution(operation_id: str, **kwargs):
     config = cloudfront.get_distribution_config(
         Id=service_instance.cloudfront_distribution_id
     )
-    config["DistributionConfig"]["ViewerCertificate"][
+    etag = config["ETag"]
+    config = config["DistributionConfig"]
```

Comment:
I'm just making sure that this intentional with re-using the variable name after it's initially set by calling the `cloudfront` API.

Reply:
yeah, it was intentional, but I could see how it might not be great. I'll push a fix real quick

Reply:
Updated :)